addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/UniFiClientThingConfig.java

```diff
@@ -41,6 +43,10 @@ public int getConsiderHome() {
         return considerHome;
     }
 
+    public boolean getBlocked() {
```

Comment:
Boolean field method should start with `is`:
```suggestion
    public boolean isBlocked() {
```

Reply:
Renamed.